partner_firstname/hooks.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import SUPERUSER_ID, api
+
+
+def post_init_hook(cr, registr):
```

Comment:
Why adding registr as second parameter and not using anymore with api.Environment.manage()?

Reply:
Update : runboat build now fails after removing the registr parameter.

2021-11-17 11:22:45,387 177 CRITICAL bf88fa468-066b-497a-95ed-00e4444d3040 odoo.service.server: Failed to initialize database `bf88fa468-066b-497a-95ed-00e4444d3040`. 
Traceback (most recent call last):
  File "/opt/odoo/odoo/service/server.py", line 1246, in preload_registries
    registry = Registry.new(dbname, update_module=update_module)
  File "/opt/odoo/odoo/modules/registry.py", line 87, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "/opt/odoo/odoo/modules/loading.py", line 474, in load_modules
    processed_modules += load_marked_modules(cr, graph,
  File "/opt/odoo/odoo/modules/loading.py", line 363, in load_marked_modules
    loaded, processed = load_module_graph(
  File "/opt/odoo/odoo/modules/loading.py", line 240, in load_module_graph
    getattr(py_module, post_init)(cr, registry)
TypeError: post_init_hook() takes 1 positional argument but 2 were given

Reply:
I'm not telling to remove it, but in the past it was _. What I miss is more the with api.Environment.manage() as first line of the method

Reply:
According to the new guidelines for migration to Odoo 15.0 :

"If you write a hook or any code that requires to create a new environment, there's no need of using the with Environment.manage(): context statement anymore."